keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -261,6 +265,35 @@ Unit, integration and E2E tests cover the existing StatefulSet mechanics.
 Additionally, unit and integration tests will be added to cover the
 API validation, behavioral change of StatefulSet with feature gate enabled and disabled.
 
+#### Prerequisite testing updates
+
+[x] I/we understand the owners of the involved components may require updates to existing tests to make this code solid enough prior to committing the changes necessary to implement this enhancement.
+
+##### Unit tests
+
+
+`k8s.io/kubernetes/pkg/apis/apps/validation`                       `06/07/2022`:    `90.6% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:93`:	   `06/07/2022`:		`93.1% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:162`:	   `06/07/2022`:		`100% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:169`:	   `06/07/2022`:		`94.1% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:199`:	   `06/07/2022`:		`100% of statements`
+`k8s.io/kubernetes/pkg/controller/statefulset`:                    `06/07/2022`:    `85.5% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset`:                 `06/07/2022`:    `76.7% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset/strategy.go:95`:	 `06/07/2022`:	  `100.0% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset/strategy.go:139`: `06/07/2022`:	  `100.0% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset/strategy.go:223`: `06/07/2022`:	  `100.0% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset/strategy.go:118`: `06/07/2022`:	  `100.0% of statements`
```

Comment:
Nit: this renders poorly if you switch to rich preview, I'd suggest wrapping this entire block in a code. Also a few words of description per each package would be nice wink You and I know what each package has and what it tests, but a sentence per each pacakge/set of packages would be nice. Also add information if you're planning to add any additional tests or the current set is sufficient.